packages/design-systems/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@tailor-platform/design-systems",
-  "version": "0.28.0",
+  "version": "0.28.1",
```

This PR now includes update on types (`MetaType`), so can you update here as the minor update?

```suggestion
  "version": "0.29.0",
```